<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana:_estadisticas</class>
 <widget class="QDialog" name="ventana:_estadisticas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estadísticas</string>
  </property>
  <widget class="QListWidget" name="lista_tiempos">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>371</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(242, 234, 220);
selection-color: rgb(255, 0, 0);
font: bold 8pt &quot;AcadEref&quot;;
color: rgb(0, 0, 0);
border-color: rgb(96, 154, 43);</string>
   </property>
  </widget>
  <widget class="QLabel" name="titulo_estadisticas">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 20pt &quot;AcadEref&quot;;</string>
   </property>
   <property name="text">
    <string>Estadísticas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_fondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 193, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regresar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>160</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgb(203, 50, 52);
	color: rgb(255,255,255);  
	border-radius: 30px; 
	border: 2px solid #000;
	font: bold 11pt &quot;AcadEref&quot;;	
}

QPushButton:hover{
	background-color: rgb(232, 3, 27);
}</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_fondo</zorder>
  <zorder>lista_tiempos</zorder>
  <zorder>titulo_estadisticas</zorder>
  <zorder>btn_regresar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
